erikiado/jp2_online
# Task/add bathroom info

#### What's this PR do?
It adds information about the bathroom in a family's house.

#### Where should the reviewer start?
Check the file familias/models.py as it has the two new fields added to the model.

#### How should this be manually tested?
Log in as capturista, and create a new study, then just that in the family information the new fields appear.

#### Any background context you want to provide?
This change has been added due to the fact that we were lacking the necessary information to fulfill the directivos requests regarding the indicators.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
